templates/openstack/managed.html

```diff
@@ -1,725 +1,618 @@
-        </div>
-      </div>
-      <div class="col-4 p-divider__block">
-        <div class="p-heading-icon--muted">
-          <div class="p-heading-icon__header">
+          <div class="p-logo-section__item">
             {{
               image(
-                url="https://assets.ubuntu.com/v1/75902002-news-feed-strip-stock_ebook.svg",
-                alt="",
-                height="28",
-                width="32",
+                url="https://assets.ubuntu.com/v1/1459d379-tmn_logo.png",
+                alt="Tele2",
+                height="130",
+                width="296",
                 hi_def=True,
-                attrs={"class": "p-heading-icon__img p-heading-icon__img--small"},
                 loading="lazy",
+                attrs={"class": "p-logo-section__logo"}
               ) | safe
-            }}
-            <h4 class="p-heading-icon__title">Ebook</h4>
+            }}      
           </div>
-          <h3 class="p-heading--four"><a class="p-link--external" href="https://ubuntu.com/engage/charmed-openstack-adoption-whitepaper" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'External Link', 'eventAction' : 'Managed cloud', 'eventLabel' : 'A guide to a successful OpenStack adoption and deployment', 'eventValue' : undefined });">A guide to a successful OpenStack adoption and deployment
-</a></h3>
-        </div>
       </div>
     </div>
   </div>
 </section>
 
-<section class="p-strip is-deep is-bordered">
-  <div class="row">
-    <div class="col-7">
-      <h2>Security first &mdash; trusted by the OpenStack majority</h2>
-    </div>
-  </div>
-  <div class="row u-equal-height">
-    <div class="col-7">
-      <p>Canonical, a founding member of OpenStack, provides security updates for thousands of production OpenStack clouds worldwide.</p>
-      <p>We offer multi-year maintenance for stable cloud deployments and upgrades for those who want to move to newer versions of OpenStack when they are released.</p>
-      <p>The OpenStack super-users who run the largest and most complex Openstack deployments choose Ubuntu because of our quality and security. Peace of mind comes from working with the people who support the widest range of cloud architectures in the widest range of business sectors and geographies.</p>
-      <p>Canonical will help you meet telco, finance and government sector audit and compliance standards, with kernel live patching, optional FIPS compliance, and a range of security and monitoring integration points.</p>
-    </div>
-    <div class="col-3 col-start-large-9">
-      {% include "openstack/shared/_openstack-pie-chart.html" %}
-    </div>
-  </div>
-</section>
-<section class="p-strip--light is-deep is-bordered">
+<section class="p-strip">
   <div class="row">
+    <h2>Why managed private cloud?</h2>
     <div class="col-8">
```

Comment:
From [Vanilla docs](https://vanillaframework.io/docs/patterns/grid):
> 
    - The page structure must be laid out using rows (.row)
    - All content must be within columns (.col-*)
    - Only columns should be direct children of a row

`u-fixed-width` is a shorthand way of expressing `.row>.col-12`, so we could get around this like this:
```suggestion
  <div class="u-fixed-width">
    <h2>Why managed private cloud?</h2>
  </div>
  <div class="row">
    <div class="col-8">
```

Reply:
Sorry didn't see this one